cg/meta/workflow/microsalt.py

```diff
@@ -59,6 +62,54 @@ def process(self) -> Process:
             )
         return self._process
 
+    def get_case_path(self, case_id: str) -> List[Path]:
+        case_obj: models.Family = self.status_db.family(case_id)
+        lims_project: str = self.get_project(case_obj.links[0].sample.internal_id)
+
+        case_path_list: List[Path] = [
+            Path(path)
+            for path in glob.glob(f"{self.root_dir}/results/{lims_project}_*", recursive=True)
+        ]
+
+        return case_path_list
+
+    def clean_run_dir(self, case_id: str, yes: bool, dry_run: bool = False):
```

Comment:
```suggestion
    def clean_run_dir(self, case_id: str, yes: bool, dry_run: bool = False) -> int:
```
or bool but I guess int is more according to the purpose

Would be great with a unit test of this one